### ISSUE TITLE
How to handle missing modalities

### ISSUE THREAD

**Issue by siemdejong** [konst-int-i/healnet#17]
Hi Hemker et al., thanks for making your HEALNet code public. I'm planning to use the model in a different setting and have a question regarding how to handle missing modalities.

Is it enough to do the following where we simply pass all available modalities to the model?
```python
from healnet.models import HealNet
from healnet.etl import MMDataset
import torch
import einops

# synthetic data example
n = 1000 # number of samples
b = 4 # batch size
img_c = 3 # image channels
tab_c = 1 # tabular channels
tab_d = 5000 # tabular features
h = 512 # image height
w = 512 # image width

tab_tensor = torch.rand(size=(n, tab_c, tab_d)) # assume 5k tabular features
img_tensor = torch.rand(size=(n, img_c, h, w)) # c h w
dataset = MMDataset([tab_tensor, img_tensor])

[tab_sample, img_sample] = dataset[0]

# batch dim for illustration purposes
tab_sample = einops.repeat(tab_sample, 'c d -> b c d', b=1)
img_sample = einops.repeat(img_sample, 'c h w -> b c (h w)', b=1)

model = HealNet(
            modalities=2, 
            input_channels=[tab_c, img_c], 
            input_axes=[1, 1], # channel axes (0-indexed)
            num_classes = 4
        )

# example forward pass, but leaving out the image sample
model([tab_sample]), model([tab_sample, img_sample])
```

Or do we have to do something special with the mask parameter of the `model.forward` method?

Either way, the output of the model seems to be the same regardless of providing one or two modalities. Is that to be expected?
```python
>>> torch.all(model([tab_sample]), model([tab_sample, img_sample]))
tensor(True)
```

**Comment by konst-int-i** [comment]
Hi @siemdejong, 

Thanks a lot for raising this issue! I am currently working on a camera-ready release which should come in the next few days. In there, we will provide some tutorials on how to run the missing modality training and inference. 

Let me know if you have any questions in the meantime. 

Thanks!

**Comment by konst-int-i** [comment]
Dear @siemdejong, 

We added all the latest changes, including a tutorial on handling the missing modalities in #22. 

I would recommend looking at the last section of the [tutorial](https://github.com/konst-int-i/healnet/blob/main/tutorial/01_Getting_Started.ipynb) for further details. 

Let me know if you encounter any further issues. 

Thanks!
